tests/unit/azure/__init__.py

```diff
@@ -11,6 +18,23 @@ def _load_fixture(filename: str):
 
 def get_az_api_mapping(*args, **_):
     mapping = _load_fixture("azure/mappings.json")[0]
+    if args[0] in mapping:
+        return mapping[args[0]]
     if args[1] in mapping:
         return mapping[args[1]]
     return {}
+
+
+def azure_api_client(mocker):
+    token = json.dumps({"aud": "foo", "tid": "bar"}).encode("utf-8")
+    str_token = base64.b64encode(token).decode("utf-8").replace("=", "")
+
+    tok = Token(access_token=f"header.{str_token}.sig")
+    mocker.patch("databricks.sdk.oauth.Refreshable.token", return_value=tok)
+    mocker.patch("uuid.uuid4", return_value="12345")
+    api_client = create_autospec(AzureAPIClient("foo", "bar"))
+    type(api_client).token = mock.PropertyMock(return_value=tok)
```

Comment:
```suggestion
    api_client.token.return_value = tok
```
This has to be a return value mock, not property mock hack. Change the production code by removing "@property" decorator if necessary